packages/components/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@terra-ui-packages/components",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "build": "tsc && tsc -p tsconfig.cjs.json",
+    "test": "jest"
+  },
+  "type": "module",
+  "module": "./lib/es/index.js",
+  "main": "./lib/cjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/es/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
+  "files": [
+    "lib/cjs/**",
+    "lib/es/**"
+  ],
+  "dependencies": {
+    "lodash": "^4.17.21",
+    "react-hyperscript-helpers": "^2.0.0"
```

Comment:
I don't see where it uses `react-hyperscirpt-helpers`.

Reply:
You're right, it doesn't. I think I intended to use it for Interactive, but then ran into a stumbling block: https://github.com/DataBiosphere/terra-ui/pull/3992#discussion_r1282437729.

Reply:
I would tend to not including it until it's actually used, but given that I was OK with leaving the test-utils dependency in even though it isn't used yet I guess it's not a big deal.

Reply:
Removed in https://github.com/DataBiosphere/terra-ui/pull/3992/commits/83b8c59498302d510c449a3303ec1e11871ab159.